packages/server/app/Controllers/Http/SearchUsersController.ts

```diff
@@ -144,17 +149,17 @@ export default class SearchUsersController {
         const user = await User.findOrFail(tmpAuthUserID);
         await user.load('devices');
         await user.load('mpeRooms');
-        const followingCounter = 12; //FIXME follow unfollow dev
-        const followersCounter = 13; //FIXME follow unfollow dev
+        await user.load('followers');
+        await user.load('following');
```

Comment:
Same comment about the user of [`user.loadCount()`](https://docs.adonisjs.com/guides/models/relationships#lazy-load-relationship-aggregates).